client/src/components/PeopleTable.tsx

```diff
@@ -182,7 +194,7 @@ export default class PeopleTable extends React.Component<TableProps, TableState>
       <Container>
         <MaterialTable
           columns={this.state.columns}
-          data={this.state.people}
+          data={this.state.data}

```

Comment:
A better approach to instead of doing the fields in render, is to abstract the render() method all together to ATable class, like following the Hollywood pattern.